pandas/types/concat.py

```diff
@@ -231,8 +231,9 @@ def union_categoricals(to_union):
     Raises
     ------
     TypeError
-        If any of the categoricals are ordered or all do not
-        have the same dtype
+        - all input do not have the same dtype
+        - all input do not have the same ordered property
+        - if all input are ordered and its categories are not identical
```

Comment:
input -> inputs,  Last line I'd maybe write as:
 - all inputs are ordered and their categories are not identical